test/test_python.py

```diff
@@ -58,6 +58,13 @@ def test_python2js(selenium):
         """)
 
 
+def test_python2js_long_ints(selenium):
+    assert selenium.run('2**30') == 2**30
+    assert selenium.run('2**31') == 2**31
```

Comment:
So this relies on the fact that in Python
```py
>>> 1. == 1
True
```
wow, I have not realized that dtype was discarded In Python for int/float comparisons..